client/src/components/DownloadModal.js

```diff
@@ -11,16 +11,17 @@ import { useDownloadModal } from 'hooks/useDownloadModal'
 
 // Button and Modal to show when downloading
 export const DownloadModal = ({
-  icon,
+  label,
   resource: initialResource,
-  publicComputedFile: initialPublicComputedFile
+  publicComputedFile: initialPublicComputedFile,
+  icon = null,
+  color = 'brand',
```

Comment:
So after looking at how we use color and label. I think we should remove color and just have the caller do something like `<DownloadModal label={<Text color="brand">My Label</Text>} />` when they want to override it? This is specifically because it leads to weird behavior if you pass a color and a label that specified a color, the `color` attribute wouldn't affect the label.

Reply:
See the commit at 50ec00cf5370b93ebaf309789055804ad0e75259.